lang/dart.md

```diff
@@ -0,0 +1,188 @@
+---
+lang: Dart
+syntax: Dart
+title:  Dart is Smart Language!
+author: chandan0000
+---
+
+```dart
+void main(){
+    print("hello world");
+}
+```
+## What is Dart?
+Dart is a new general and open source programming language with a vibrant community developed by Google Inc. and its official website is https://dart.dev. It was first announced as a public preview on October 10, 2011; it has now reached version 1.10. World class language designers and developers are involved in this project, namely, Lars Bak and Kasper Lund (known for their V8 JavaScript engine embedded in the Chrome browser, which revolutionized performance in the JavaScript world), and Gilad Bracha (a language theorist known for the development of the Strongtalk and Newspeak languages and for the Java specification). Judging by the huge amount of resources and the number of teams working on it, it is clear that Google is very serious about making Dart a success.
+
+## What is Dart used for?
+Dart programming language can be used for web and mobile applications and is also suitable for server and desktop applications.
+
+Dart wasn’t popular and was known as an orphan, but it had a compiler for a Dart VM, also Google had all the control about this language.
+
+## Why to use Dart?
+- Dart is really flexible
+- Dart embraced the open source ecosystem
+- Dart is pretty easy to learn
+- Dart is a good first language
+- Dart has great tooling support
+- Dart is a robust language
+- Dart ensures productivity
+- Dart is used by Flutter
+- Dart is backed by a tech giant
+- Dart solved a lot of problems, efficiently
+
+## Tools You Will Need
+For setting up environment for performing java codes you need with a minimum of 2 GB of RAM.
+You will also need the following softwares :
+- Linux, macOS or Windows xp/7/8/10 operating system
+- Dart SDK
+- Text Editors
+
+
+## Setting up Dart development environment
+**Executing Script Online with DartPad:**
+
+You may test your scripts online by using the online editor at https://dartpad.dartlang.org/. The Dart Editor executes the script and displays both HTML as well as console output. The online editor is shipped with a set of preset code samples.
+
+
+
+**Dart Installation:**
+ 
+ 
+**Install Dart on Windows:**
+In this tutorial, we will provide a step by step process to get Dart installed on Windows.
+
+Dart SDK comes as a pre-compiled version. So, download and extracting is all you need to do.
```

Comment:
```suggestion
In this tutorial, we will provide a step-by-step process to get Dart installed on Windows.</br>
Dart SDK comes as a pre-compiled version. So, download and extracting is all you need to do.
```

This will make it look like

In this tutorial, we will provide a step-by-step process to get Dart installed on Windows.
Dart SDK comes as a pre-compiled version. So, download and extracting is all you need to do.